crates/driver/src/boundary/settlement.rs

```diff
@@ -212,7 +213,7 @@ impl Settlement {
                 })
                 .collect(),
         )?;
-        let gas_price = eth::U256::from(auction.gas_price().effective()).to_big_rational();
+        let gas_price = eth::U256::from(gas_price.effective()).to_big_rational();
```

Comment:
Wouldn't it make sense to keep the gas_price on the auction object accurate (ie account for the extra MEVBlocker tip)? This way solvers will also receive the "correct" gas price in the instance in case they compute their own scores.